tests/bucketlist_db_size_check.sql

```diff
@@ -0,0 +1,17 @@
+{{ config(
+    severity="error"
+    , tags=["singular_test"]
+    )
+}}
+
+with bucketlist_db_size as (
+  select sequence,
+    closed_at,
+    total_byte_size_of_bucket_list / 1000000000 as bl_db_gb
+  from {{ source('crypto_stellar', 'history_ledgers') }}
```

Comment:
I think this should be a `ref('stg_history_ledgers')` instead of a source. Otherwise these tests would be hardcoded to just prod right?

Reply:
Wont test still use `crypto_stellar.history_ledgers` instead of `test_crypto_stellar.history_ledgers` ?

Example: https://github.com/stellar/stellar-dbt-public/blob/2e63a8a7a00937213372d0ab636016373d59f94b/models/staging/stg_account_signers.sql#L6-L28

Reply:
It will use the test project/dataset because the source for staging tables will be overwritten by the dbt_project.yml in the private dbt repo
https://github.com/stellar/stellar-dbt/blob/master/dbt_project.yml#L63-L68
```
+project: "{% if target.name == 'prod' %} crypto-stellar {% else %} {{ target.project }} {% endif %}"
```

Reply:
Got it. Yes, agree in that case we should just use ref

Reply:
Addressed in https://github.com/stellar/stellar-dbt-public/pull/83/commits